GitVersionCore/Configuration/Wizard/SetBranchTag.cs

```diff
@@ -0,0 +1,54 @@
+namespace GitVersion
+{
+    using System.Collections.Generic;
+
+    public class SetBranchTag : ConfigInitWizardStep
+    {
+        string name;
+        readonly BranchConfig branchConfig;
+
+        public SetBranchTag(string name, BranchConfig branchConfig)
+        {
+            this.branchConfig = branchConfig;
```

Comment:
You use name / branchconfig everywhere (right order), make it the same please :-)